src/components/central-program/program-data-overview-table.js

```diff
@@ -139,25 +141,25 @@ const BudgetOverview = ({ data }) => {
     const budget_delta = data[CHANGE][BUDGET]
 
     rows.push({
-      description: "Change in budget from previous year",
+      description: columnsByDataField[`${CHANGE}.${BUDGET}`].displayName,
       value: `${deltaPrefix(budget_delta)}${formatToUSD(
         Math.abs(budget_delta)
       )}`,
     })
   }
 
   let overOrUnder = "Over",
-    difference
+    balance
   if (data[SPENDING] > data[BUDGET]) {
-    difference = data[SPENDING] - data[BUDGET]
+    balance = data[SPENDING] - data[BUDGET]
   } else {
     overOrUnder = "Under"
-    difference = data[BUDGET] - data[SPENDING]
+    balance = data[BUDGET] - data[SPENDING]
   }
 
   rows.push({
-    description: "Over or under budget?",
-    value: `${overOrUnder} by ${formatToUSD(difference)} (${Math.abs(
+    description: columnsByDataField[BALANCE].displayName,
+    value: `${overOrUnder} by ${formatToUSD(balance)} (${Math.abs(
```

Comment:
"Over/Under by " here isn't being translated, maybe add an overOrUnder property columnsByDataField.budget_balance?

Reply:
thanks. yeah, I need to tidy that up. wasn't exactly sure what the translation should be but I'll pick something and can always come back and adjust.